src/GridCal/Engine/Simulations/ShortCircuitStudies/short_circuit.py

```diff
@@ -53,20 +55,20 @@ def short_circuit_unbalance(bus_idx, Z0, Z1, Z2, Vbus, Zf, fault_type):
     Zth2 = Z2[bus_idx, bus_idx]
     Zflt = Zf[bus_idx]
 
-    if fault_type == 'LG':
+    if fault_type == FaultType.LG:
         I0 = Vpr / (Zth0 + Zth1 + Zth2 + 3 * Zflt)
         I1 = I0
         I2 = I0
-    elif fault_type == 'LL':  # between phases b and c
+    elif fault_type == FaultType.LL:  # between phases b and c
         I0 = 0
         I1 = Vpr / (Zth1 + Zth2 + Zflt)
         I2 = - I1
-    elif fault_type == 'LLG':  # between phases b and c
+    elif fault_type == FaultType.LLG:  # between phases b and c
         I1 = Vpr / (Zth1 + Zth2 * (Zth0 + 3 * Zflt) / (Zth2 + Zth0 + 3 * Zflt))
         I0 = -I1 * Zth2 / (Zth2 + Zth0 + 3 * Zflt)
         I2 = -I1 * (Zth0 + 3 * Zflt) / (Zth2 + Zth0 + 3 * Zflt)
     else:
-        print('Unknown fault type')
+        print('Unknown unbalanced fault type')
```

Comment:
Raise exception since it is certain that the type is unknown

Reply:
Done